app/cells/decidim/proposals/vote_proposal_cell.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    class VoteProposalCell < Decidim::ViewModel
```

Comment:
```suggestion
module Decidim
  module DecidimAwesome
    module Votings
      class ThreeFlagsProposalCell < Decidim::ViewModel
```
We need to isolate this voting method system. In case we add others (that we will)